package.json

```diff
@@ -9,8 +9,6 @@
   },
   "homepage": "open.inf.is",
   "repository": "OpenINF/openinf.github.io",
-  "license": "MIT OR Apache-2.0 OR BlueOak-1.0.0",
```

Comment:
unlike our software packages, this is to be a full-fledged website w/ a ton of content, much of which is not similarly licensed to this